src/Parser/HolyPriest/CPM_ABILITIES.js

```diff
@@ -15,7 +15,8 @@ const CPM_ABILITIES = [
     spell: SPELLS.PRAYER_OF_MENDING_CAST,
     name: 'Prayer of Mending',
     category: SPELL_CATEGORY.ROTATIONAL,
-    getCooldown: (haste, combatant) => (12 - (combatant.hasTalent(SPELLS.PIETY_TALENT.id) ? 2 : 0)) / (1 + haste),
+    getCooldown: (haste, combatant) => ((12 - (combatant.hasTalent(SPELLS.PIETY_TALENT.id) ? 2 : 0)) + 1.5) / (1 + haste), // +1.5 for base cast time
+    recommendedCastEfficiency: 0.70, // There's a lot of things that can potentially delay PoM: movement, heavy damage, etc. It is important to maintain high uptime but delaying a few GCDs is not bad.
```

Comment:
This still does not seem to function correctly for including the cast time. For my recent Logs I reversed engineered the math and it is calculating my casts for the fight off the CD. My CD for PoM is 8.1 Seconds, on a 3:06 second fight it reports I should have 23 Casts or 186 / 8.1 = 22.96, Really with the cast time of about 1.2 seconds it should be 186 / 9.3 or 20 casts at 65% Effic